etc/contracts-test-data/contracts/loadnext/loadnext_contract.sol

```diff
@@ -26,17 +43,25 @@ contract LoadnextContract {
             sum += readArray[i];
         }
 
-        for (uint i = 0; i < writes; i++) {
+        for (uint i = 0; i < newWrites; i++) {
             writeArray.push(i);
         }
 
+        uint arrayLength = writeArray.length;
+        require(arrayLength > 0);
+        for (uint i = 0; i < overWrites; i++) {
+            writeArray[i % arrayLength] = i;
+        }
+
```

Comment:
so during the run of loadtest it will first deploy the contract and then keep writing to the same slots among different transactions. I think it's fine.. But let's compare the number of real initial and repeated writes in the DB to the requested ones, to make sure the ratio holds.